fe/src/project/AspNetCore/shared/components/Header/Header.js

```diff
@@ -93,6 +93,15 @@ function Header(props) {
             setSearchTerm(newValue);
         }
     };
+
+    useEffect(() => {
```

Comment:
**suggestion:** let's get rid of it completely

Reply:
Without this, our quantity count show after refresh. I made it, to make updates of totalBasketItems after click "Dodaj do koszyka"